### ISSUE TITLE
Improving the "README" understanding for non-technical users 

### ISSUE THREAD

**Issue by Nickel14** [kelszo/obsidian-file-explorer-plus#15]
### What would you like to be added?

First thanks you for your plugin, very useful!! 
I suggest to rebuild the "ReadMe" file to improve the readability and the understandability of the plugins.

I'm not a technician and for me it's been very difficult to understand how to use some features of the plugin. Now i understand more, but still something is not clear.
For example, the modifier of the filter "Strict, Wildcard and Regex" what are they, and how to use it? 
Or another example, how to use properly a "Tag patter". 

Maybe isn't a priority issue, but to broaden the use of the plugin to non-technicians users, it could be a usefull thing.

Thanks again for your work

### Why is this needed?

A remaking of the "readMe" file, to improve the understandibility and the readability.

**Comment by kelszo** [comment]
Yes this is a great idea, I will try to rewrite the README. Most likely I will include some examples.
